### PR TITLE
Add option to disable protobuf generation

### PR DESCRIPTION
**Pull request by saschagrunert**
Refers to https://github.com/knqyf263/go-plugin/issues/67

Testing it in https://github.com/containerd/nri/pull/121

### CODE REVIEW

File: gen/main.go [hunk 1]
```diff
@@ -165,14 +173,16 @@ func (gg *Generator) generatePBFile(f *fileInfo) {
...
+		}
+		for _, enum := range f.allEnums {
+			genEnum(g, f, enum)
+		}
+		for _, message := range f.allMessages {
+			genMessage(g, f, message)
+		}
 	}
 	for _, service := range f.allServices {
 		genServiceInterface(g, f, service)
```
Comment (saschagrunert) [inline gen/main.go:188]: we still need to generate the service interface

**Comment by saschagrunert** [comment]
https://github.com/containerd/nri/pull/121 compiles now locally.

**Comment by saschagrunert** [comment]
@knqyf263 PTAL

**Review by knqyf263 — COMMENTED** [review]
Thanks for your contribution! Could you also update README? It should mention how to use go-plugin with `protoc-gen-go` with an example command.

**Comment by saschagrunert** [comment]
> Thanks for your contribution! Could you also update README? It should mention how to use go-plugin with `protoc-gen-go` with an example command.

Sure, I updated the README as suggested.

**Review by knqyf263 — APPROVED** [review]

**Comment by knqyf263** [comment]
Thanks

**Pull request merged**